### PR TITLE
140: update login and registration pages

### PR DESCRIPTION
**Pull request by IlliaLop**

### CODE REVIEW

File: frontend/src/app/modules/authentication/registration/registration.component.sass [hunk 1]
```diff
@@ -18,6 +18,11 @@
         font-size: 24px
         @extend .title
 
+.input
+    height: 92px
```
Comment (serhiiyasenev) [inline frontend/src/app/modules/authentication/registration/registration.component.sass]: looks like a duplicate - probably can be moved to styles or shared?

**Review by serhiiyasenev — DISMISSED** [review]

File: frontend/src/app/shared/components/input/input.component.sass [hunk 2]
```diff
@@ -48,3 +48,9 @@ input:focus
     img
         margin-bottom: -4px
         width: 24px
+        height: 24px
+
+.hide img[src$='hide-1.svg']
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/input/input.component.sass]: Don't use src here...
What it if will be changed
Create additional class if needed

**Review by OlegKarapysh — DISMISSED** [review]

**Comment by alinaSielina** [comment]
please add required changes

**Comment by IlliaLop** [comment]
> please add required changes

Yeah, I'm back to this branch now) I was focusing on another one

File: frontend/src/app/shared/components/input/input.component.html [hunk 3]
```diff
@@ -11,8 +11,9 @@
                 'padding-right': type === 'password' ? '40px' : '15px'
             }"/>
         <span *ngIf="type === 'password'" class="hide" (click)="show = !show">
-            <img src="assets/{{ show ? 'hide-1.svg' : 'hide-2.svg' }}" alt="show/hide"/>
+            <img [src]="show ? 'assets/hide-1.svg' : 'assets/hide-2.svg'" alt="show/hide" [ngStyle]="{'width.px': show ? 20 : 24, 'height.px': show ? 20 : 24, 'margin-right.px': show ? 2 : 0, 'margin-bottom.px': -4 }" />
```
Comment (alinaSielina) [inline frontend/src/app/shared/components/input/input.component.html]: please create 2 classes

Comment (serhiiyasenev) [inline frontend/src/app/shared/components/input/input.component.html]:
```
<img [src]="show ? 'assets/hide-1.svg' : 'assets/hide-2.svg'" alt="show/hide" [ngStyle]="{'width.px': show ? 20 : 24, 'height.px': show ? 20 : 24, 'margin-right.px': show ? 2 : 0, 'margin-bottom.px': -4 }" />
```
what happens here? 😄

**Review by alinaSielina — APPROVED** [review]

**Review by serhiiyasenev — APPROVED** [review]

**Pull request merged**